container/container.go

```diff
@@ -61,22 +61,23 @@ type ExitStatus struct {
 type Container struct {
 	StreamConfig *stream.Config
 	// embed for Container to support states directly.
-	*State          `json:"State"` // Needed for Engine API version <= 1.11
-	Root            string         `json:"-"` // Path to the "home" of the container, including metadata.
-	BaseFS          string         `json:"-"` // Path to the graphdriver mountpoint
-	RWLayer         layer.RWLayer  `json:"-"`
-	ID              string
-	Created         time.Time
-	Managed         bool
-	Path            string
-	Args            []string
-	Config          *containertypes.Config
-	ImageID         image.ID `json:"Image"`
-	NetworkSettings *network.Settings
-	LogPath         string
-	Name            string
-	Driver          string
-	OS              string
+	*State             `json:"State"` // Needed for Engine API version <= 1.11
+	Root               string         `json:"-"` // Path to the "home" of the container, including metadata.
+	BaseFS             string         `json:"-"` // Path to the graphdriver mountpoint
+	RWLayer            layer.RWLayer  `json:"-"`
+	ID                 string
+	Created            time.Time
+	Managed            bool
+	Path               string
+	Args               []string
+	Config             *containertypes.Config
+	ImageID            image.ID `json:"Image"`
+	ImageVariantDigest string
```

Comment:
Some discussion around the name (bike shed!); let's give people some time to comment alternatives 👍  (also a comment, and perhaps an explicit `json:...`)

Reply:
As `ImageVariantDigest` isn't obvious in a vacuum/outside this diff, I'd prefer something like `ImageResolvedRef` myself.